src/core/server/saved_objects/saved_objects_service.ts

```diff
@@ -144,6 +145,11 @@ export interface SavedObjectsServiceSetup {
    * ```
    */
   registerType: <Attributes = any>(type: SavedObjectsType<Attributes>) => void;
+
+  /**
+   * Returns the default index used for saved objects.
```

Comment:
Do plugins need it to read from the index? If so, let's add a warning that plugins shouldn't write directly to the index